srcpkgs/marvin/template

```diff
@@ -0,0 +1,56 @@
+# Template for Marvin
+pkgname=marvin
+version=18.16.0
+revision=1
+maintainer="Brenton Horne <brentonhorne77@gmail.com>"
+homepage="https://chemaxon.com/products/marvin"
+license="marvin"
+depends="oracle-jdk"
+short_desc="Skeletal structure drawing program by ChemAxon"
+only_for_archs="x86_64"
+repository=nonfree
+restricted=yes
+nostrip=yes
+
+_baseUrl="https://dl.chemaxon.com/marvin/"
+_filename="marvin_linux_$(echo $version | sed 's/.[0-9]*$//g').deb"
+_marvinUrl="https://dl.chemaxon.com/marvin/${version}/${_filename}"
```

Comment:
Why use a additional variable when it only gets used once, you could put the content directly into distfiles.

Reply:
OK shall do that too, thanks.